test/verify/machineslib.py

```diff
@@ -1113,6 +1118,32 @@ def installWithErrorTest(dialog):
                                                    os_name=config.MACOS_X_LEOPARD,
                                                    connection='session'))
 
+        # Start of tests for import existing disk as installation option
+        createTest(TestMachines.VmDialog(self, "subVmTestCreate16", sourceType='disk_image',
+                                         location=config.VALID_DISK_IMAGE_PATH,
+                                         memory_size=256, memory_size_unit='MiB',
+                                         os_vendor=config.OPENBSD_VENDOR,
+                                         os_name=config.OPENBSD_5_4,
+                                         start_vm=False))
+
+        # Recreate the image the previous test just deleted to reuse it
+        self.machine.execute("qemu-img create {0} 500M".format(TestMachines.TestCreateConfig.VALID_DISK_IMAGE_PATH))
+
+        # Unload KVM module, otherwise we get errors getting the nested VMs
+        # to start properly.
+        # This is applicable only for the following test so let's keep it last,
+        # in order to allow the rest of the tests to run faster with QEMU KVM
+        # Run modprobe -r in retry loop because it fails sometimes with 'Module kvm is in use'
+        self.machine.execute("for i in 1 2 3 4 5; do modprobe -r kvm_intel && modprobe -r kvm && break || sleep 1; done")
```

Comment:
Style nitpick for the future: `machine.execute()` already has a builtin timeout, and there is no reason to be overly parsimonious about it. Also, for this we have `testlib.wait()`:

    wait(lambda: self.machine.execute("modprobe -r kvm_intel && modprobe -r kvm"))